### ISSUE TITLE
Unable to receive or publish video to video room - VideoBridgeView

### ISSUE THREAD

**Issue by arsalanj** [Igor-Khomich/JanusAudioStreamPlayer#6]
Using the example I couldn't get the Video Bridge to work.
I have provided the base URL in the Environment, which is being connected successfully, I can see the Janus initialization event with a list of rooms, once I connected to the desired room I get the event of 'joined'.

The app is connected to the web(web app is also being used through Janus for video call) but I am unable to see the video stream of the remote participant, also in the web app, I cant see my mobile feed either.

However, I am getting all the events, the web app is also getting all the events, its just the streams are not working for me, both audio and video.

**Comment by Igor-Khomich** [comment]
Hi @arsalanj, did you figure out how to deal with your problem?

**Comment by arsalanj** [comment]
No, https://janus.conf.meetecho.com/janus/ this URL seems to be working, Video is being sent to this but I am unable to receive any video.
Secondly, we have tried to make our own signaling server using Janus, when I give the URL of that in the environment file the above problems still persist.
Janus API's seem to be working fine but not getting any video (both on device and on web, however, web to web connections are fine)

**Comment by arsalanj** [comment]
I closed the issue since it was working with metecho URL, so most probably there is some sort of problem with our own server, I am thinking certificate or SSL related.

**Comment by arsalanj** [comment]
![image](https://user-images.githubusercontent.com/11611707/106106078-d0a58900-6166-11eb-9c27-9333747fe527.png)

I get json decode error this error as well when I try to join the room

**Comment by Igor-Khomich** [comment]
@arsalanj this should not be a problem, the server sends a lot of different events, and I parse only a few that I need, the rest are displayed as an "error". Can you share the console log to me?

**Comment by updev0612** [comment]
I also has such problems.

**Comment by soring323** [comment]
hello. Igor. I'm trying to implement video room function with Janus. 
And found this repo. It's very helpful for me. 
It is working well to join to exist streaming session(channel) but we have problem to implement video bridge to join to video room. 
Console. log is like this. 
EVENT HAVE COME: {
   "janus": "keepalive"
}
SendLongPollEventsHandler started
EVENT HAVE COME: {
   "janus": "keepalive"
}
SendLongPollEventsHandler started

when click join button with room id 1234(I guess u used fixed room Id in project)
webRTCClient didChangeConnectionState!!!
"dataChannel did change state: RTCDataChannelState"
EVENT HAVE COME: {
   "janus": "webrtcup",
   "session_id": 101574157392009,
   "sender": 8758540262046228
}
SendLongPollEventsHandler started
EVENT HAVE COME: {
   "janus": "media",
   "session_id": 101574157392009,
   "sender": 8758540262046228,
   "type": "audio",
   "receiving": true
}
SendLongPollEventsHandler started
EVENT HAVE COME: {
   "janus": "media",
   "session_id": 101574157392009,
   "sender": 8758540262046228,
   "type": "video",
   "receiving": true
}
SendLongPollEventsHandler started
